Website/src/rendering/src/Layout.tsx

```diff
@@ -35,12 +35,19 @@ const Layout = ({ context }: LayoutProps): JSX.Element => {
     context.pageState === LayoutServicePageState.Preview
       ? 'experience-editor-active'
       : '';
+  const contextTitle = context['EventInfo'] as NodeJS.Dict<string | string>;
+  let pageTitle = contextTitle.titlePrefix;
+  if (route?.fields?.pageTitle?.value) {
+    pageTitle += ' - ' + route?.fields?.pageTitle?.value;
+  } else {
+    pageTitle += ' - Page';
```

Comment:
I do not think it is still needed to add " - Page" to the page title. "PLAY! Summit" would be good enough for pages without a page title.